doc/source/examples/incompressible-flow/3d-flow-over-periodic-hills/3d-flow-over-periodic-hills.rst

```diff
@@ -0,0 +1,280 @@
+
+.. warning:: It is important to remember that the output path folder, in this case ``output`` must exist before running the simulation.
+
+
+Physical properties
+~~~~~~~~~~~~~~~~~~~
+
+The physical properties section is used to target a specific Reynolds number:
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Physical Properties
+    #---------------------------------------------------
+    subsection physical properties
+        set kinematic viscosity        = 1.78571E-04 # Re = 5600
+    end
+
+Recall the definition of the Reynolds number:
+
+.. math::
+ Re = \frac{u_B h}{\nu}
+
+Taking this into account and the fact that the height of the hill :math:`h` is 1 :math:`m`, we set the bulk velocity to 1 :math:`m s^{-1}` and the ``kinematic viscosity`` parameter to ``1.78571E-04`` to obtain a Reynolds number of 5600. 
+
+Mesh 
+~~~~~
+
+The mesh subsection specifies the computational grid:
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Mesh
+    #---------------------------------------------------
+    subsection mesh
+        set type                       = periodic_hills
+        set initial refinement         = 5
+        set grid arguments 		       = 1;1;4;2;1
+    end
+
+The standard geometry is included in the Lethe code with 6 polynomials designing the curve of the hill. Therefore the ``type`` parameter is set to ``periodic_hills``. It consists of a static, uniformly refined mesh with isoparametric elements. The ``initial refinement`` parameter does a uniform refinement in the triangulation. The ``grid argument`` parameter requires five values that correspond to ``spacing y lines; alpha; repetitions x; repetitions y; repetitions z``. A brief explanation of each of them is given here:
+
+* ``spacing y lines``: the periodic hill grid is generated with equally spaced horizontal lines or with gradually spaced lines which means more lines near upper and lower walls. This parameter may be set in the range from 0 to 1, with 0 meaning equally spaced lines and 1 the maximum of shifting lines.
+
+* ``alpha``: While the geometry of the benchmark is fixed, it can be elongated to compare the behavior of the flow with the slope. Therefore, this parameter elongates the slopes of the geometry but keeps the length of the flat region. This parameter should be set in the range from 0.5 to 3, where 1 has no effect on the geometry.
+
+* ``repetitions x, repetitions y, repetitions z``: To get cells with an aspect ratio different on the domain, different numbers of subdivisions are set, given by repetition in different coordinate directions. The minimum number of subdivisions in each direction is 1. 
+
+The following image displays a coarse mesh for this example. It can be seen that the horizontal lines are shifted with the associated parameter to get more lines near walls. Here, repetitions for x, y, z allow the mesh to get more cells in x and y directions.
```

Comment:
I think this chunk should definitely be in the parameters/mesh.rst file as there is still no mention of the periodic hill! And maybe substitute here with a paragraph that explains the parameter choice if needed.

Reply:
Could be in both really :)

Reply:
Very good idea Laura